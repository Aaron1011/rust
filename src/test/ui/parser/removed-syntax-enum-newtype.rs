enum e = isize; //~ ERROR expected `where`, or `{` after enum name, found `=`
