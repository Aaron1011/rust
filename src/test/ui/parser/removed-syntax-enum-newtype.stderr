error: expected `where`, or `{` after enum name, found `=`
  --> $DIR/removed-syntax-enum-newtype.rs:1:8
   |
LL | enum e = isize;
   |        ^ expected `where`, or `{` after enum name

error: aborting due to previous error

